driver_unix.go

```diff
@@ -4,11 +4,27 @@ package continuity
 
 import (
 	"fmt"
+	"os"
 	"sort"
+	"syscall"
 
 	"github.com/stevvooe/continuity/sysx"
 )
 
+const (
+	atNoFollow = 0x100
+)
+
+func (d *driver) Unlink(path string) error {
+	return syscall.Unlink(path)
+}
+
+// Lchmod changes the mode of an file not following symlinks. The name
+// must be an absolute path to a file.
+func (d *driver) Lchmod(name string, mode os.FileMode) error {
+	return syscall.Fchmodat(0, name, uint32(mode), atNoFollow)
```

Comment:
This is not a correct use of `Fchmodat`, if I understand correctly. This will do the operation relative to the file descriptor in the first argument.